controllers/common.go

```diff
@@ -29,6 +29,33 @@ import (
 	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	INVALID_CREDENTIALS_EVENT = "InvalidCredentials"
```

Comment:
As discussed, can you prefix these with EVENT_ rather than using it as a suffix to maintain consistency and improve readability?

Reply:
Done!